src/runtime_src/core/common/api/aie/xrt_graph.cpp

```diff
@@ -298,6 +313,11 @@ graph(const xrt::device& device, const xrt::uuid& xclbin_id, const std::string&
   : handle(open_graph(device, xclbin_id, name, am))
 {}
 
+graph::
+graph(const xrt::hw_context& hwctx, const std::string& name, graph::access_mode am)
+    : handle(open_graph(hwctx, name, am))
```

Comment:
Might as well fold in the otherwise unused static open_graph(hwctx, ...)
```suggestion
    : handle{std::make_shared<xrt::graph_impl>(hwctx, name, am)}
```